crates/swc_ecma_codegen/src/text_writer/basic_impl.rs

```diff
@@ -44,29 +44,26 @@ impl<'a, W: Write> JsWriter<'a, W> {
         }
     }
 
-    fn write_indent_string(&mut self) -> io::Result<usize> {
-        const INDENT: &[u8] = b"    ";
+    fn write_indent_string(&mut self) -> Result {
+        const INDENT: &str = "    ";
 
-        let mut cnt = 0;
         for _ in 0..self.indent {
-            cnt += self.raw_write(INDENT)?;
+            self.raw_write(INDENT)?;
         }
 
-        Ok(cnt)
+        Ok(())
     }
 
-    fn raw_write(&mut self, data: &[u8]) -> io::Result<usize> {
-        let written = self.wr.write(data)?;
```

Comment:
Since the `Result<usize>` was never used, the code here assumed that the data would always be written to the output. This would be a bug if not all the data was written, so I changed it to use `write_all`.